Packages/com.unity.inputsystem/InputSystem/Plugins/NPad/NPadSupport.cs

```diff
@@ -0,0 +1,70 @@
+namespace UnityEngine.Experimental.Input.Plugins.Switch
```

Comment:
Breaks build outside of editor on players other than Switch. I'll fix it after the merge.